docs/time/index.rst

```diff
@@ -936,17 +940,15 @@ dynamical models). An example using the JPL ephemerides is:
 .. doctest-requires:: jplephem
 
     >>> ltt_bary_jpl = times.light_travel_time(ip_peg, ephemeris='jpl') # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    >>> ltt_bary_jpl # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    <TimeDelta object: scale='tdb' format='jd' value=[-0.0037715  -0.00377286]> # doctest: +REMOTE_DATA +IGNORE_OUTPUT
```

Comment:
Here and in the line below, replace `+IGNORE_OUTPUT` by `+FLOAT_CMP` (might as well test it!).